src/typegraph/specs/codegen/tests/utils.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+const typeAliasCase = `
```

Comment:
thought: we'll have to be careful with what features of typescript we exercise. Future target langs might not be as expressive.